python/src/integry/tests/test_apps.py

```diff
@@ -0,0 +1,96 @@
+import unittest
+import os
+from integry import Integry
+from datetime import datetime
+
+APP_KEY = os.getenv("INTEGRY_APP_KEY")
+APP_SECRET = os.getenv("INTEGRY_APP_SECRET")
+USER_ID = os.getenv("INTEGRY_USER_ID")
+
+class TestIntegryFunctionAPI(unittest.IsolatedAsyncioTestCase):
```

Comment:
```suggestion
class TestIntegryAppsAPI(unittest.IsolatedAsyncioTestCase):
```

Let's use "app" instead of "functions" in this file